bindings/python/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+set(PYTHON_COMPONENTS Development.Module)
```

Comment:
I have to add this line here, otherwise I have a cmake error saying [Could NOT find Python3 (missing: Python3_LIBRARIES Development](https://github.com/fabinsch/proxsuite/actions/runs/4344064366/jobs/7586888432#step:7:255).

Reply:
I don't understand why cmake on conda on windows behave like that. You can start by checking if this cmake is correctly >= 3.18, where `Development.Module` was added, otherwise I would just add this `if(UNIX)`

Reply:
The cmake version in the conda env is [3.25.2](https://github.com/Simple-Robotics/proxsuite/actions/runs/4344208280/jobs/7587235447#step:7:51), so I ended up doing the `if(UNIX)`.